python/requirements/nsight_mock/setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
```

Comment:
if it will be just a simple python file, you do not need to use setup.py. you can just:

- add `#!/usr/bin/env python3` at the start of the file.
- copy the python file to `/usr/local/bin`, and drop the `.py` suffix.

Reply:
I'll check that out